examples/jdbc-client-crud-operations/jdbc_client_crud_operations.bal

```diff
@@ -45,7 +45,7 @@ public function main() {
     // of the `java.jdbc` module.
     int age = 24;
     string name = "Anne";
-    ret = testDB->update("INSERT INTO student(age, name) values (?, ?)",
+    ret = testDB->update("INSERT INTO student(age, name) values (?, ?)", false,
```

Comment:
Why we can't remove passing `false`? we already have set `false` as default value

Reply:
If update statement which doesn't have parameters, no need to append false but In this case, update statements have parameters So we need to set that value.